process/templates/PROJECT_NAME_incubation_stage.md

```diff
@@ -0,0 +1,33 @@
+## Project incubation application
+
+### List of project maintainers
+The project must have a minimum of three maintainers with a minimum of two different company affiliations.
```

Comment:
Suggest "organizational" instead of "company" for reasons discussed before.

Reply:
Since PR#117 is landed, should we update and resolve this?